product-service/addProduct.js

```diff
@@ -0,0 +1,65 @@
+const { Client } = require('pg');
+
+const { successResponse, unexpectedErrorResponse, validationErrorResponse, ProductExistsErrorResponse } = require('helper');
+const { dbOptions } = require('./db/config');
+const { validateCount, validatePrice, validateTitle, validateDescription } = require('./validators');
+const { ValidationError, ValidationNotPresentError } = require('./errors');
+
+function validateBody(body) {
+    const { title, price, count, description } = body;
+    if(title === undefined) throw new ValidationNotPresentError('title');
+    if(price === undefined) throw new ValidationNotPresentError('price');
+    if(count === undefined) throw new ValidationNotPresentError('count');
+    if(description === undefined) throw new ValidationNotPresentError('description');
```

Comment:
u could use npm module e.g. `yup`
it can validate models by created schema as:
```
 const schema = Yup.object().shape({
    id: Yup.string().uuid(),
    title: Yup.string().required(),
    description: Yup.string(),
    price: Yup.number().required(),
    count: Yup.number().required(),
  })

schema.validate(body)
  ```